1-python-question.py

```diff
@@ -23,13 +23,50 @@
     "applicants": ["Amy", "Ally", "David", "Brendan", "Zoho"]
 }
 
+def CheckBannedVisitors(applicant):
+    for banned_visitor in banned_visitors:
+        return banned_visitor == applicant
 
-def processRequest(request):
-    # Your code here
-    return
+def CheckSuccessfulApplicants(applicant):
+    for banned_visitor in banned_visitors:
+        return banned_visitor != applicant
```

Comment:
These 2 functions will not return the correct values. You just happened to get lucky in the ordering of the applicants and banned applicants :P